hats/details.md

```diff
@@ -2,32 +2,52 @@
-5. Each Hat behaves as if it were upgraded semi-independently, with new features/settings that match specific needs:
+4. If a user accepts the suggestion of the Suggested Tile, this Tile becomes a Permanent Tile;
+5. Clicking on either the Suggested Tile or the Permanent Tile switches to a *Firefox Hat*;
+6. A *Firefox Hat* is basically an instance of Firefox window with its own visual theme and configured specifically for one website or a collection of websites;
+7. Each Hat behaves as if it were upgraded semi-independently, with new features/settings that match specific needs:
   * As Firefox gains new WebVR features, Gaming-oriented Hats receive a Suggested Tile that offers users the ability to turn on experimental features;
-  * As Firefox gains new privacy protection features, Hats that already have some Privacy settings on turn have these features turned on automatically;
-  * Some new privacy protection features are developped as add-ons, which appear as Suggested Tiles for Privacy-aware hats;
+  * As Firefox gains new privacy protection features, Privacy-oriented Hats turn have these features turned on automatically;
+  * Individual Hats can receive further features that are not turned on automatically but appear as Suggested Tiles;
   * ...
-6. To switch between Hats, the user may use the Window menu, or a shortcut. Additionally, opening a tab using a bookmark, or the search bar, or typing a url, causes the appropriate Hat to be picked;
-   _(or the current Hat if no Hat can be determined? Or should that be the Vanilla Hat?)_
-7. Tabs can be moved between Hats with drag/drop or by right-clicking on a tab and selecting `Send To...`. Menu `Send To...` contains the list of all Hats that have been created, as well as a submenu `New...` offering specific templates (e.g. Gaming, Banking, Blank) for manually creating Hats;
+8. Several tabs can be active at once. To switch between Hats, the user may use:
+  * the Permanent Tile;
+  * the Window menu (even if the Hat hasn't been loaded yet);
+  * a keyboard shortcut;
+  * the OS's process-switching mechanism (e.g. Ctrl-Tab, the Dock, etc.);
+  * the icon in the Start Menu, the Applications Menu, etc;
+  * a built-in page `about:hats`;
+9. Also, following a link, opening a bookmark, typing a URL for a websites that is associated to a Hat switches to that Hat, while following a link/bookmark/URL for a website that is not associated switches back to the Vanilla Hat;
+10. Tabs can be moved between Hats with drag/drop or by right-clicking on a tab and selecting `Send To...`. Menu `Send To...` contains the list of all Hats that have been created, as well as a submenu `New...` offering specific templates (e.g. Gaming, Banking, Blank) for manually creating Hats. Bookmarks can be moved between Hats using essentially the same mechanism;
 
 Variant workflow
 ================
 
-1. User visits getfirefox.com and downloads a Hat.
+1. User visits getfirefox.com and pick a Hat in the collection of Hats.
```

Comment:
Maybe more concretely: user visits getfirefox.com/secure and get a download link to the Secure Firefox Hat/installer.  Now if someone searches for "secure online banking" we have a landing page just for them.